- It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is  the group significantly different (with respect to systolic blood pressure!)  from the regular population?

- Set up the hypothesis test.
- Write down all the steps followed for setting up the test.
- Calculate the test statistic by hand and also code it in Python. It should be # 4.76190. What decision can you make based on this calculated value?

In [2]:
from scipy import stats
import numpy as np
import pandas as pd

population mean: μ = 120

sample size : n = 100 

sample mean: x̅ = 130.1

Standard Deviation sample: σn = 21.21

### Two-Sided Test: Hypotheses 

H0: μ = x̅ (The group is not significantly different from the rest of the population)

Ha: μ != x̅ (The group is significantly different from the population)

### Test Statistic (t) [calculated by hand]

t = (130.1 - 120)/((21.21)/sqrt(100))
t = 10.1/2.121

t = 4.76


### Confidence Level 

95% -> significance level α = 0.025 (α = 0.05/2 because it´s a two-sided test)


### T-table Critical Value (tc)
for df = 100-1 = 99 and α = 0.025

tc = 1.984 


In [3]:
#Test Statistic (t) [using python]

sample_mean = 130.1
pop_mean = 120
sample_std_dev = 21.21

t = ( sample_mean - pop_mean ) / ( sample_std_dev / np.sqrt(100) )
print("Our t score is: {:.2f}".format(t))

Our t score is: 4.76


In [4]:
# T-table Critical Value (tc) [using python]

tc = +-stats.t.ppf(1-(0.05/2),df=99)
tc

-1.9842169515086827

### Result

t = 4.76 > +-1.98 =tc

t is larger than tc

The H0 can be rejected, because the group is significantly different from the population.


In a packing plant, a machine packs cartons with jars. It is supposed that a new machine will pack faster on the average than the machine currently used. To test that hypothesis, the times it takes each machine to pack ten cartons are recorded. The results, in seconds, are shown in the tables in the file Data/machine.txt. Assume that there is sufficient evidence to conduct the t test, does the data provide sufficient evidence to show if one machine is better than the other


In [10]:
New Machine	  Old Machine
42.1	        42.7
41	            43.6
41.3	        43.8
41.8	        43.3
42.4	        42.5
42.8	        43.5
43.2	        43.1
42.3	        41.7
41.8	        44
42.7	        44.1

SyntaxError: invalid syntax (1310423556.py, line 1)

In [16]:
list = open('Data/machine.txt').read()

FileNotFoundError: [Errno 2] No such file or directory: 'Data/machine.txt'